src/shared/actions/challenge-listing/sidebar.js

```diff
@@ -138,6 +138,8 @@ export default createActions({
 
       SAVE_FILTER: saveFilter,
 
+      SAVING_FILTER_BEGIN: _.noop,
```

Comment:
@daft300punk Have not tried, though, looking through the code, it looks fine. But... please rename these actions into `SAVE_FILTER_INIT` and `SAVE_FILTER_DONE` to be consistent with naming of other action pairs for async actions.